src/config/model.rs

```diff
@@ -215,6 +217,14 @@ impl Config {
             .map(|sig| sig.to_owned())
     }
 
+    pub fn signature_separator(&self, account: &Account) -> Option<String> {
```

Comment:
In fact a signature separator fn does not make sense (it will never be used alone). It makes more sense *I think* to move this part inside the `signature` fn. What do you think?

Reply:
Yeah maybe. The only thing I'm wondering if handling defaults here is desired? Cause I think if it's moved into `signature` then the two `Optional<String>`s need to be merged somehow, either by replacing them by defaults if `None` or returning `(Option<String>, Option<String>)`. What do you think?

Reply:
Indeed, I did not think about this. After all, maybe the simplest way is to include the separator *inside* the signature. The user needs to add the separator *inside* his signature (if he wants one). It just needs to be better documented. What do you think?

Reply:
Ah yes, this is probably the way to go :) I just tried that one can indeed do multiline strings in the config 👍

Reply:
Removed the option again. I made the example config in the README use a multiline signature, do you think that's useful?

Reply:
I added an explanation in the wiki (didn't know anyone could edit it :))

Reply:
Oh great! Thank you :smiley: